tests/networks/nets/test_milmodel.py

```
@@ -81,11 +81,11 @@ def test_ill_args(self):
                 mil_mode="att_trans_pyramid",
             )
 
-    def test_script(self):
+    def test_export(self):
         input_param, input_shape, expected_shape = TEST_CASE_MILMODEL[0]
```

Ruff (0.15.6)

[warning] Unpacked variable `expected_shape` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)